src/main/java/com/pusher/client/crypto/nacl/SecretBoxOpener.java

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2015 Eve Freeman
```

Comment:
should this one have the copyright even though bits of it have been edited?

Reply:
I guess what i'm more trying to say is, should we highlight the bits which you've changed?

Reply:
it is a "substantial portion of the Software", indeed most of the stuff here is copied. I added though Pusher's copyright as well: 451a3f232fbcb1aa6359fa2b69368c1ae149171f
The whole SDK is also licensed on MIT so easy compatibility-wise.